packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -271,30 +269,50 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
       throw new Error('undefined fee')
 
     // We have to round because coinselect library uses sats per byte which cant be decimals
-    const fast = String(Math.round(feeData.data.fast?.satsPerKiloByte / 1024))
-    const average = String(Math.round(feeData.data.average?.satsPerKiloByte / 1024))
-    const slow = String(Math.round(feeData.data.slow?.satsPerKiloByte / 1024))
+    const fastPerByte = String(Math.round(feeData.data.fast?.satsPerKiloByte / 1024))
+    const averagePerByte = String(Math.round(feeData.data.average?.satsPerKiloByte / 1024))
+    const slowPerByte = String(Math.round(feeData.data.slow?.satsPerKiloByte / 1024))
```

Comment:
?. can result in undefined, so what does the output look like when that is the case? It should be easy to add test cases to cover that scenario

Reply:
There is a check above which should narrow and not require the optional chaining here. This is a scenario that may be better moved into the unchained api as well where we throw if fees are not correctly returned from the node

Reply:
good point, got rid of the optional chaining